### PR TITLE
Function to get the frequency of the hybrid MECO

### PR DESCRIPTION
**Pull request by micamu**
Add the hybrid energy from https://arxiv.org/pdf/1602.03134.pdf to pnutils and compute the velocity and the frequency of the hybrid MECO

### CODE REVIEW

File: pycbc/pnutils.py [hunk 1]
```diff
@@ -797,3 +797,141 @@ def t2_cutoff_frequency(m1, m2, chi1, chi2):
...
+        Quadrupole-monopole term of the secondary object (1 for black holes).
+
+    Returns
+    -------
+    h_E: float
+        The hybrid energy as a function of v
+    """
+
+    pi = numpy.pi
+
```
Comment (cdcapano) [inline pycbc/pnutils.py]: Define variables for v**2, v**4, chi1**2, etc. here, so you don't recompute them in each term; e.g., `v4 = v**4`, then use `v4` everywhere below.

**Comment by micamu** [comment]
I always said it was a very long equation!
Is there a way to see the ci/sideci results without having to give it access to your personal data?

**Comment by titodalcanton** [comment]
IMO the big equation should go in lalsimulation as a C function...

**Comment by micamu** [comment]
@titodalcanton We know I have to code it up in lalsimulation. But we want to use it now, and since I am not very familiar with lal it was going to be faster to just do it here temporarily...

**Comment by micamu** [comment]
@cdcapano I made the changes you requested.
For the example case you showed me with positive spins and hybrid MECO below Schwarzschild ISCO: I do not get the same result in Mathematica. Python gives a velocity of 0.391, Mathematica gives me 0.481 (Schwarzschild ISCO is at 0.4089). Let me check what is happening and then I will push the changes.

**Comment by micamu** [comment]
@cdcapano The last commit has the new version of the energy we worked on last week. I have tried that it works on a few values, but it would be helpful if you could start your inspiral PE run with this cutoff to see if it still gives some errors...

**Comment by micamu** [comment]
I don't know what to do with codacy and landscape, they complain about trailing whitespaces, blank lines and having too many arguments (I use as many as needed!) and too many local variables (Collin asked me to define those local variables for the v**2, v**3, etc).

**Comment by spxiwh** [comment]
I figured I would use this as a test of the "edit a pull request directly" functionality. I'm fixing the whitespace issues.

**Comment by micamu** [comment]
Thanks @spxiwh! I do like to leave an empty line between the doc and the actual function, but I guess if landscape doesn't like it I am not allowed to do it...

**Comment by cdcapano** [comment]
Looks good. I tested this in a PE run and it worked fine for all waveforms. Speed is must better now; about ~1ms per waveform rather than the ~7ms it was before. That's still slow-ish for TaylorF2, but as Tito said, the energy equation should eventually be moved to lalsimulation, which will speed it up further.

**Pull request merged**